frontend/lang/messages/en-US.json

```
@@ -872,7 +872,8 @@
     "you-are-offline-description": "Not all features are available while offline. You can still add, modify, and remove items, but you will not be able to sync your changes to the server until you are back online.",
     "are-you-sure-you-want-to-check-all-items": "Are you sure you want to check all items?",
     "are-you-sure-you-want-to-uncheck-all-items": "Are you sure you want to uncheck all items?",
-    "are-you-sure-you-want-to-delete-checked-items": "Are you sure you want to delete all checked items?"
+    "are-you-sure-you-want-to-delete-checked-items": "Are you sure you want to delete all checked items?",
+    "no-shopping-list-found": "No Shopping Lists Found."
```

I think this looks better without the period, since it's standalone text. Also want to make sure the key matches the text

```suggestion
    "no-shopping-lists-found": "No Shopping Lists Found"
```